geoportal/geoportailv3_geoportal/static-ngeo/js/offline/lux-offline/lux-offline-reload.component.ts

```diff
@@ -0,0 +1,66 @@
+import 'jquery';
+import 'bootstrap/js/modal.js';
+import i18next from 'i18next';
+import {LuxBaseElement} from '../../LuxBaseElement';
+import {html} from 'lit';
+import {customElement, state, query} from 'lit/decorators.js';
+import { LuxOfflineServiceInstance } from './lux-offline.service';
+
+@customElement('lux-offline-reload')
+export class LuxReloadAlert extends LuxBaseElement {
+
+  @state()
+  private status;
+
+  private prevStatus;
+
+  @query('.modal')
+  private modal: HTMLElement;
+
+  constructor() {
+    super();
+    //this.offlineService = new LuxOfflineService();
+    this.prevStatus = LuxOfflineServiceInstance.status$.getValue();
+    LuxOfflineServiceInstance.status$.subscribe((status)=> {
+      if (status != 'IN_PROGRESS' && this.prevStatus != undefined
+          && this.prevStatus != 'NO_INIT'
+        && status != this.prevStatus) {
+        $(this.modal).modal('show');
```

Comment:
Uuh, not easy to follow this condition, but I guess it's correct. I'd use the `enum` values and strict checking (`!==`) anyhow.